commonutils/publish.gradle

```diff
@@ -44,6 +44,13 @@ publish.dependsOn 'assembleRelease'
 
 publishing {
 
+    // use `./gradlew publish` to deploy the artifact locally for testing
+    repositories {
+        maven {
+            url "../repo"
```

Comment:
The artifact directory is inside the project directory so that its easier to inspect/cleanup/etc. I will add a comment to indicate that publishing to mavenLocal is also possible via the `publishToMavenLocal` task.